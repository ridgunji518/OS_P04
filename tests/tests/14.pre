./edit-makefile.sh ../solution/Makefile test_14 > ../solution/Makefile.test
cp -f tests/test_14.c ../solution/test_14.c
cd ../solution/
make clean
make -f Makefile.test clean
make -f Makefile.test xv6.img
make -f Makefile.test fs.img
cd ../tests

