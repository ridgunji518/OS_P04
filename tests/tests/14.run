cd ../solution; ../tests/run-xv6-command.exp CPUS=4 Makefile.test test_14 | grep -E 'XV6_TEST_OUTPUT|XV6_TEST_ERROR'; rm *test_14* ;cd ../tests
