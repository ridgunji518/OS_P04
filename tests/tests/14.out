XV6_TEST_OUTPUT After malloc with THP enabled, 1048575B - program base pages:257 huge pages:0
XV6_TEST_OUTPUT After malloc with THP enabled, 200B - program base pages:265 huge pages:0
XV6_TEST_OUTPUT After malloc with THP enabled, 1MB - program base pages:265 huge pages:1
XV6_TEST_OUTPUT After malloc with THP enabled, 2MB - program base pages:265 huge pages:1
XV6_TEST_OUTPUT After malloc with THP enabled, 8KB - program base pages:265 huge pages:1
XV6_TEST_OUTPUT After malloc with THP enabled, 1MB - program base pages:265 huge pages:2
XV6_TEST_OUTPUT After malloc with THP enabled, 4MB - program base pages:265 huge pages:3
XV6_TEST_OUTPUT After malloc with THP enabled, less than 1MB - program base pages:265 huge pages:3
